Orange/evaluation/scoring.py

```diff
@@ -249,6 +249,14 @@ def compute_score(self, results):
         return np.sqrt(MSE(results))
 
 
+class CVRMSE(Score):
+    """
+    Coefficient of variation of the RMSE
```

Comment:
Wouldn't it be better to call this normalized RMSE? I know [it is ambiguous](https://en.wikipedia.org/wiki/Root-mean-square_deviation), but CVRMSE doesn't tell me anything.